dbflute-runtime/src/main/java/org/dbflute/infra/doc/decomment/DfDecoMapFile.java

```diff
@@ -105,6 +107,7 @@
     //     ; databaseComment = sea mystic
     //     ; commentVersion = 1
     //     ; authorList = list:{ cabos ; hakiba ; deco ; jflute }
+    //     ; branchName = master
```

Comment:
Does the branchName mean name of branch for current piece?
For example, authors are saved as history by authorList.
Does'nt the branch name have it?

If it's so, pieceBranch...now thinking.

Reply:
made it!